inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/metric/SinkMetricData.java

```diff
@@ -286,16 +286,17 @@ public void invokeDirty(long rowCount, long rowSize) {
     @Override
     public String toString() {
         return "SinkMetricData{"
-                + "groupId='" + groupId + '\''
-                + ", streamId='" + streamId + '\''
-                + ", nodeId='" + nodeId + '\''
-                + ", numRecordsOut=" + numRecordsOut.getCount()
-                + ", numBytesOut=" + numBytesOut.getCount()
-                + ", numRecordsOutForMeter=" + numRecordsOutForMeter.getCount()
-                + ", numBytesOutForMeter=" + numBytesOutForMeter.getCount()
-                + ", dirtyRecords=" + dirtyRecords.getCount()
-                + ", dirtyBytes=" + dirtyBytes.getCount()
-                + ", numRecordsOutPerSecond=" + numRecordsOutPerSecond.getRate()
-                + ", numBytesOutPerSecond=" + numBytesOutPerSecond.getRate()
+                + "metricGroup=" + metricGroup
+                + ", labels=" + labels
+                + ", auditImp=" + auditImp
+                + ", numRecordsOut=" + numRecordsOut
+                + ", numBytesOut=" + numBytesOut
+                + ", numRecordsOutForMeter=" + numRecordsOutForMeter
+                + ", numBytesOutForMeter=" + numBytesOutForMeter
+                + ", dirtyRecords=" + dirtyRecords
+                + ", dirtyBytes=" + dirtyBytes
+                + ", numRecordsOutPerSecond=" + numRecordsOutPerSecond
+                + ", numBytesOutPerSecond=" + numBytesOutPerSecond
```

Comment:
Counter and Meter don't override toString(). So this print info is not useful. It need print getCount() and getRate().